pyartifactory/models/repository.py

```diff
@@ -114,6 +115,11 @@ class ContentSynchronisation(BaseModel):
     source: Source = Source()
 
 
+class FederatedMembers(BaseModel):
+    url: str = ""
+    enabled: bool = False

```

Comment:
From what I can see in the [Artifactory API](https://jfrog.com/help/r/jfrog-rest-apis/repository-configuration-json), enabled is a str, not a boolean.

```
"members": [{"url": "http://targetartifactory/artifactory/repositoryName", "enabled":"true"},
```

Reply:
this is true. The "problem" is: the repsonse object returns this field as boolean.
So basically there are two ways:
1. keep it as it is. It'S not 100% clean but works as the api seems to accept a boolean too
2.  model an additional reposonse model:
   ```
   class FederatedMembers(BaseModel):
    url: str = ""
    # enabled: bool = True
    enabled: str = "true"

class FederatedMembersResponse(BaseModel):
    url: str = ""
    enabled: bool = True
    ```
    This seems a little bit weird as it's only nessecary because of an unclean design of the api

Reply:
I would go for option 2. And to ensure people write either "true" or "false", I would add a literal:
```
enabled: Literal["true", "false"] = "true"
```